include/mbedtls/ssl.h

```diff
@@ -185,6 +185,9 @@
 #define MBEDTLS_SSL_PRESET_DEFAULT              0
 #define MBEDTLS_SSL_PRESET_SUITEB               2
 
+#define MDEDTLS_SSL_SUPPRESS_CA_LIST_ENABLED     1
```

Comment:
Please avoid double negatives. Make the constants e.g. `MBEDTLS_SSL_SEND_CA_LIST` and `MBEDTLS_SSL_OMIT_CA_LIST`. Or a boolean “send CA list?” if defaulting to a nonzero representation is acceptable for binary compatibility.